### PR TITLE
Add tranditional Chinese language

### PR DESCRIPTION
**Pull request by Lafudoci**

### CODE REVIEW

**Comment by m2049r** [comment]
@Lafudoci the folder name should be ```values-zh-rTW``` (notice the small 'r').

**Review by m2049r — APPROVED** [review]

**Review by erciccione — APPROVED** [review]
Reviews of the content should come soon

**Comment by Lafudoci** [comment]
@hjt00000 @LuPoYi @bcexecd @qxxrbull 你們願意幫忙看看翻譯內容有沒有什麼問題嗎?

File: app/src/main/res/values-zh-rTW/strings.xml [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+
+    <string name="info_txdetails_xmrto_key">金鑰:</string>
+    <string name="info_txdetails_xmrto_address">地址:</string>
+    <string name="info_txdetails_xmrto_status">狀態:</string>
+    <string name="info_txdetails_xmrto_amount">%1$s BTC</string>
+
+
+    <string name="info_send_xmrto_paid">等待確認中</string>
+    <string name="info_send_xmrto_unpaid">等待付款中</string>
+    <string name="info_send_xmrto_error">XMR.TO錯誤 (%1$s)</string>
```
Comment (qxxrbull) [inline app/src/main/res/values-zh-rTW/strings.xml]: XMR.TO發生錯誤 (%1$s)

Comment (qxxrbull) [inline app/src/main/res/values-zh-rTW/strings.xml]: XMR.TO發生錯誤 (%1$s)

Comment (qxxrbull) [reply app/src/main/res/values-zh-rTW/strings.xml]: 其他應該沒啥問題

Comment (Lafudoci) [reply app/src/main/res/values-zh-rTW/strings.xml]: 感謝你，已修改!

File: app/src/main/res/values-zh-rTW/help.xml [hunk 2]
```diff
@@ -0,0 +1,173 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="help_create_new"><![CDATA[
+        <h1>建立全新的錢包</h1>
+        <p>如果你需要一個新的Monero錢包地址!</p>
+        <p>請輸入一個獨特的錢包名稱與錢包密碼。
+        這個密碼是用來保護你在這個裝置上的錢包資料，請使用足夠強度的密碼。</p>
+        <h2>請抄寫你的記憶種子碼!</h3>
```
Comment (hjt00000) [inline app/src/main/res/values-zh-rTW/help.xml]: `<h2>請抄寫你的記憶種子碼!</h3> --> </h2>`
不過我看原始的似乎就給錯，不知道正確的是什麼？

**Review by hjt00000 — CHANGES_REQUESTED** [review]
簡單提供小瑕疵XD

File: app/src/main/res/values-zh-rTW/help.xml (hunk 2 above)
Comment (m2049r) [reply app/src/main/res/values-zh-rTW/help.xml]: 是的，事实证明是错误的。 谢谢！

Comment (Lafudoci) [reply app/src/main/res/values-zh-rTW/help.xml]: @hjt00000 感謝你眼睛好利XD! 我這邊也一起修正了。

File: app/src/main/res/values-zh-rTW/help.xml [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+        <p>若你知道你的第一筆交易是發生在哪個區塊高度的話，請在欄位\"回復高度\"中
+        輸入數值 - 若此欄留空則將會掃描<em>整個</em>區塊鏈以尋找屬於你錢包的交易與
+        餘額，將會花費<em>較長</em>的時間。</p>
+    ]]></string>
+
+    <string name="help_details"><![CDATA[
+        <h1>錢包詳細資訊</h1>
+        <h2>公開地址</h2>
+        你的公開地址就像是銀行帳號一樣可以與他人分享而不需要擔心
+        會失去你的，其他人可以透過這個地址發送Monero到你的錢包。
```
Comment (0140454) [inline app/src/main/res/values-zh-rTW/help.xml]: 「會失去你的，...」
這邊需要加個Monero嗎？ 原文是「losing your Monero.」

File: app/src/main/res/values-zh-rTW/help.xml [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+        <p>這個設定將會影響你的交易被納入區塊的速度。越高優先權的交易將會花費更多的手續費，反之亦然。
+        請注意若你選擇低手續費則可能會讓你的交易等待數小時才能被納入區塊。預設優先權為\"中等\"。</p>
+        <h1>發送BTC</h1>
+        <h2>XMR.TO</h2>
+        <p>XMR.TO 是一個可以將XMR轉換為BTC的第三方服務。
+        我們利用XMR.TO的API將其服務整合至Monerujo中。請參閱 https://xmr.to 後自行決定是否使用。
+        Monerujo開發團隊與XMR.TO並無相關，無法為你解決其服務上的問題</p>
+        <h2>XMR.TO 匯率<h2>
+        <p>在 \"金額\" 的頁面中你將看到 XMR.TO 服務的參數。包含了匯率以及BTC金額的上下限。
+        請注意這個階段顯示的匯率並不是確定的。你也可以看到多少金額的可以進行零確認XMR的BTC發送(詳見 XMR.TO的FAQ以了解更多)。請注意XMR.TO並不額外收取手續費用，是不是很棒呢?</p>
```
Comment (0140454) [inline app/src/main/res/values-zh-rTW/help.xml]: 「你也可以看到在多少金額之下可以直接發送BTC，而不用等待XMR的確認。」
這樣翻不知道會不會更好？

File: app/src/main/res/values-zh-rTW/help.xml [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+<resources>
+    <string name="help_create_new"><![CDATA[
+        <h1>建立全新的錢包</h1>
+        <p>如果你需要一個新的Monero錢包地址!</p>
+        <p>請輸入一個獨特的錢包名稱與錢包密碼。
+        這個密碼是用來保護你在這個裝置上的錢包資料，請使用足夠強度的密碼。</p>
+        <h2>請抄寫你的記憶種子碼!</h2>
+        <p>在下一個步驟將會顯示你的25字 \"記憶種子碼\"。
+        這是你唯一需要用來回復錢包的資訊。
+        並且可以用來取用錢包的完全權限。
```
Comment (0140454) [inline app/src/main/res/values-zh-rTW/help.xml]: 「這是你在未來想回復錢包並且取得完整權限時所需的唯一資訊。」
這樣翻不知道會不會更好？

**Review by 0140454 — COMMENTED** [review]
小小的想法

File: app/src/main/res/values-zh-rTW/help.xml (hunk 3 above)
Comment (Lafudoci) [reply app/src/main/res/values-zh-rTW/help.xml]: 對是我漏寫了XD，謝謝勘誤，已修改!

File: app/src/main/res/values-zh-rTW/help.xml (hunk 4 above)
Comment (Lafudoci) [reply app/src/main/res/values-zh-rTW/help.xml]: 感謝建議，確實會更通順，已修改!

File: app/src/main/res/values-zh-rTW/help.xml (hunk 5 above)
Comment (Lafudoci) [reply app/src/main/res/values-zh-rTW/help.xml]: 感謝建議，確實會更通順，已修改!

File: app/src/main/res/values-zh-rTW/about.xml [hunk 6]
```diff
@@ -0,0 +1,628 @@
...
+    <string name="credits_text"><![CDATA[
+        <b>致謝</b>
+        <br/>
+        m2049r, baltsar777, anhdres, keejef,
+        rehrar, EarlOfEgo et al.
+        <br/><br/>
+        <a href="https://monerujo.io/">monerujo.io</a>
+        ]]></string>
+
+    <string name="privacy_policy"><![CDATA[
```
Comment (m2049r) [inline app/src/main/res/values-zh-rTW/about.xml:17]: is there a reason why the privacy policy is not translated?

Comment (Lafudoci) [reply app/src/main/res/values-zh-rTW/about.xml:17]: I worried about the translation quality and wording might bring some responsibility issue in policy and license content. I have little experience here so I left them in original text. Do you have suggestion?

Comment (Lafudoci) [reply app/src/main/res/values-zh-rTW/about.xml:17]: I found @0140454 have translated it, haha. If @m2049r doesn't think that's an issue I'll include [the PR](https://github.com/Lafudoci/xmrwallet/pull/1/files). 0140454 made a great translation work on that.

Comment (m2049r) [reply app/src/main/res/values-zh-rTW/about.xml:17]: licenses i wouldn't touch, but the privacy policy should be fine. please include it. thank you!!

Comment (Lafudoci) [reply app/src/main/res/values-zh-rTW/about.xml:17]: OK, I have reviewed @0140454 's work and merged.

**Pull request merged**